plugins/services/src/js/reducers/serviceForm/Volumes.js

```diff
@@ -75,8 +75,11 @@ module.exports = {
             break;
         }
 
-        return [].concat(this.externalVolumes,
-          this.localVolumes.filter(filterHostVolumes.bind(this)).map(mapLocalVolumes));
+        return [].concat(
```

Comment:
```JS
/**
 * This is concatinating the contents of localVolumes and externalVolumes
 * we need to filter localVolumes for hostVolumes if no docker is available that 
 * is happening in the FilterHostVolumes method. This is because in the ui local
 * Volumes and Host Volumes are represented in the same space.
 * Because both Volumes are in the same reducer we also need to transform them
 * to either a localVolume or a hostVolume because they both have a different format.
 * External Volumes is just external volumes.
 */
```

I hope this is good enough.